src/Heading/index.js

```diff
@@ -47,6 +47,7 @@ const createHeading = (
       {!isHeading1 && <Anchor id={id} />}
       <HeadingTag
         {...props}
+        id={`${id}`}
```

Comment:
The HTML spec says ids must be document-wide unique.

Reply:
Thanks @icaraps. I’ll make A change to ensure that they are.